dev_guide/builds/triggering_builds.adoc

```diff
@@ -27,7 +27,10 @@ triggers are available:
 
 Webhook triggers allow you to trigger a new build by sending a request to the
 {product-title} API endpoint. You can define these triggers using
-https://developer.github.com/webhooks/[GitHub webhooks] or Generic webhooks.
+https://developer.github.com/webhooks/[GitHub],
```

Comment:
we need to add link: in front of all links to external websites per our [doc guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references) for example:

    link:https://developer.github.com/webhooks/[GitHub]